pkg/virt-launcher/virtwrap/network/dhcp/dhcp_test.go

```diff
@@ -244,5 +245,23 @@ var _ = Describe("DHCP", func() {
 			}))
 			Expect(options[240]).To(Equal([]byte("private.options.kubevirt.io")))
 		})
+
+		table.DescribeTable("Should not set DHCP Options with invalid values", func(optionCode dhcp4.OptionCode) {
+			ip := net.IP{}
```

Comment:
You're using this `ip` as both the router IP and the client's mask. It becomes a bit confusing.

Furthermore, this leads me to questiom: how can this function work with an empty client's mask ? Shouldn't we make sure that never happens ?

Reply:
It's not empty, `DefaultMask()` returns /24 for for `net.IP{}`
https://tip.golang.org/src/net/ip.go?s=6792:6836 (and line 256)

Reply:
Oh wait, it's totally empty
`net.IP{}.To4()` does return nil, so the mask is really empty.
I'll add similar change and a test case for mas as well

Reply:
Done